internal/services/postgres/postgresql_server_resource_test.go

```diff
@@ -381,6 +385,59 @@ func TestMinTlsVersionOnServerUpdate(t *testing.T) {
 	})
 }
 
+func TestAccPostgreSQLServer_writeOnlyPassword(t *testing.T) {
+	data := acceptance.BuildTestData(t, "azurerm_postgresql_server", "test")
+	r := PostgreSQLServerResource{}
+
+	resource.ParallelTest(t, resource.TestCase{
+		TerraformVersionChecks: []tfversion.TerraformVersionCheck{
+			tfversion.SkipBelow(version.Must(version.NewVersion("1.11.0-beta1"))),
```

Comment:
Should this be 1.11.0?

```suggestion
			tfversion.SkipBelow(version.Must(version.NewVersion("1.11.0"))),
```